sdk/apimanagement/Azure.ResourceManager.ApiManagement/src/autorest.md

```diff
@@ -273,6 +273,13 @@ rename-mapping:
   ParameterContract.required: IsRequired
   SchemaType: ApiSchemaType
   ApiRevisionContract.privateUrl: privateUrlString
+  ApiContract.properties.termsOfServiceUrl: termsOfServiceLink
+  ApiContract.properties.serviceUrl: serviceUrlLink
+  ApiUpdateContract.properties.termsOfServiceUrl: termsOfServiceLink
+  ApiUpdateContract.properties.serviceUrl: serviceUrlLink
+  ApiCreateOrUpdateParameter.properties.termsOfServiceUrl: termsOfServiceLink
+  ApiCreateOrUpdateParameter.properties.serviceUrl: serviceUrlLink
```

Comment:
I think in dotnet we should use `uri` instead of `url`.
Or we might need to remove the word `uri` in these names, because in some cases, their values do not look like a uri at all (Is this true?)